### PR TITLE
Use file scoped namespaces

### PR DESCRIPTION
**Pull request by jnyrup**
Upgrading to C# 10 unlocks file scoped namespaces.
Changed about every file except a few test classes that has dummy types inside the same file that needs to be in a specific/different namespace.

### CODE REVIEW

File: Tests/FluentAssertions.Specs/AssertionOptionsSpecs.cs [hunk 1]
```diff
@@ -10,395 +10,391 @@
 using Xunit;
 using Xunit.Sdk;
 
-namespace FluentAssertions.Specs
+namespace FluentAssertions.Specs;
```
Comment (jnyrup) [inline Tests/FluentAssertions.Specs/AssertionOptionsSpecs.cs:13]: Changed namespace from `FluentAssertions.Specs.AssertionOptionsSpecs` to `FluentAssertions.Specs`

File: Tests/FluentAssertions.Specs/Formatting/EnumerableExtensionsSpecs.cs [hunk 2]
```diff
@@ -1,34 +1,31 @@
 ﻿using System.Collections.Generic;
-
 using FluentAssertions.Formatting;
-
 using Xunit;
 
-namespace FluentAssertions.Specs
+namespace FluentAssertions.Specs.Formatting;
```
Comment (jnyrup) [inline Tests/FluentAssertions.Specs/Formatting/EnumerableExtensionsSpecs.cs:5]: Changed namespace from `FluentAssertions.Specs` to `FluentAssertions.Specs.Formatting`.

File: Tests/FluentAssertions.Specs/AssertionOptionsSpecs.cs [hunk 3]
```diff
@@ -10,395 +10,391 @@
...
+
+public class AssertionOptionsSpecs
 {
-    namespace AssertionOptionsSpecs
```
Comment (jnyrup) [inline Tests/FluentAssertions.Specs/AssertionOptionsSpecs.cs:15]: Changed namespace from `FluentAssertions.Specs.AssertionOptionsSpecs` to `FluentAssertions.Specs`

**Review by dennisdoomen — APPROVED** [review]
You're on a roll ;-)

**Pull request merged**